example/spec/serial/AsyncOperationRunnerFailingLambdaSpec.js

```diff
@@ -30,12 +28,6 @@ describe('The AsyncOperation task runner executing a failing lambda function', (
       asyncOperationsTableName = `${config.stackName}-AsyncOperationsTable`;
       failFunctionName = `${config.stackName}-AsyncOperationFail`;
 
-      asyncOperationModel = new AsyncOperation({
```

Comment:
You should be able to delete this on line 28 as well

```
asyncOperationsTableName = `${config.stackName}-AsyncOperationsTable`;
```

Reply:
Hm, I don't think I can yet but this highlights that we can probably refactor ecs().runTask